sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ReceiverManager.cs

```diff
@@ -123,6 +121,8 @@ public virtual async Task ReceiveAndProcessMessagesAsync(CancellationToken cance
             }
         }
 
+        public virtual Task Cancel() => Task.CompletedTask;
```

Comment:
nit: I'd suggest still calling this method CancelAsync so it is clear that it may be performing an async call.